app/models/user.rb

```diff
@@ -72,10 +72,7 @@ class User < ActiveRecord::Base
   scope :visible,         -> { except_hidden }
   scope :completer_scope, ->(opts) { visible }
 
-  validates :mail, :format => { :with => /\A(([\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~]+((\.\"[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+(\.[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+)*\")*\.[\w!#\$%&\'\*\+\-\/=\?\^`\{\|\}~]+)*)|(\"[\w !#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+(\.[\w !#\$%&\'\*\+\-\/=\?\^`\{\|\}~\"\(\),:;<>@\[\\\] ]+)*\"))
-                                          @[a-z0-9]+((\.[a-z0-9]+)*|(\-[a-z0-9]+)*)*\z/ix },
-                   :length => { :maximum => 254 },
-                   :allow_blank => true
+  validates :mail, :email => true, :allow_blank => true
```

Comment:
length validation is no longer present, is that on purpose? I don't think the regexp catches it